src/strategy.coffee

```diff
@@ -13,19 +13,25 @@ module.exports = (oAuth2Strategy) ->
       options.authorizationURL ?= constants.defaultAuthorizationURL
       options.tokenURL ?= constants.defaultTokenURL
       options.verifyURL ?= constants.defaultVerifyURL
+      @params = new Object()
+      @params.scope = options.scope or ''

```

Comment:
Why create a new object?  How about 
```coffeescript
@_scope = options.scope or ''
```
Also, instance variables meant to be private should be prefaced with a '_'